src/app/app.component.ts

```diff
@@ -1,10 +1,66 @@
-import { Component } from '@angular/core';
+import {Component, OnInit} from '@angular/core';
+import {GitHubUsersService} from './git-hub-users.service';
+import {GitHubUsers} from './git-hub-users';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css']
 })
-export class AppComponent {
-  title = 'angular-training';
+export class AppComponent implements OnInit{
+  // title of the component
+  public title = 'angular-training';
+
+  // store of github users
+  public gitHubUsers = '';
+  private gitHubUsersHolder: Array<GitHubUsers>;
+
+  // the number of users to present
+  N_USERS = 5;
+
+  // constructor populating private variables
+  constructor(private gitHubUserService: GitHubUsersService) {}
+
+  /**
+   * converts the array of git hub users into a strong for presenting
+   * @private
+   */
+  private convertToString(): void {
+      // go over the users and convert to the string
+      const tabSpace = '\t\t\t';
```

Comment:
Of course this only works because we're using a `<pre>` element that behaves a bit like a terminal console. If you were to write `\t` in any other element it would render as those two chars `\` and `t`.

Reply:
ah. guess id need more pad starts and stops to get them aligned instead to be better safe.